src/algebra/order/floor.lean

```diff
@@ -575,6 +575,8 @@ begin
     exact ⟨y, hys, rfl⟩ }
 end
 
+lemma fract_mem (x : α) : int.fract x ∈ Icc (0:α) 1 := ⟨int.fract_nonneg _, (int.fract_lt_one _).le⟩
```

Comment:
This lemma feels a bit weird to me; I would expect to have the `Ico` variant, and only the `Icc` variant as well if actually need it
```suggestion
lemma fract_mem_Ico (x : α) : int.fract x ∈ Ico (0 : α) 1 := ⟨int.fract_nonneg _, int.fract_lt_one _⟩
lemma fract_mem_Icc (x : α) : int.fract x ∈ Icc (0 : α) 1 := ⟨int.fract_nonneg _, (int.fract_lt_one _).le⟩
```